general-configuration-guide/attribution-requirements.md

```diff
@@ -12,14 +12,14 @@ There is a **POWERED BY TINY** attribution in the status bar that is enabled by
 When including attribution, the following requirements must be met:
 
 * "**POWERED BY TINY**" to be stated or the [{{site.companyname}} logo]({{site.url}}/guidelines/#logo) placed in a readily visible location whenever the editor is loaded.
-* The word "**Tiny**" or the {{site.companyname}} logo should have a hyperlink to {{site.url}}.
+* The word "**Tiny**" or the {{site.companyname}} logo should have a hyperlink to [{{site.url}}]({{site.url}}).
 * The location of the product attribution is to be:
 
   * In the right-hand corner of the status bar of the {{site.productname}} editor; or
   * An alternate location adjacent to the {{site.productname}} editor that is no less prominent.
 
-The editor demo on the [{{site.companyname}} homepage]({{site.url}}) displays the status bar product attribution as in the screenshot below:
+For example, the following example shows the status bar product attribution in the bottom, right-hand corner:
```

Comment:
Sounds a little weird having `example` repeated twice here, but I can't think of a better way to reword it :shrug:

Reply:
just remove 'For example, '?

Reply:
as an example

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1503/commits/96d49d5bb465257353154955fc3d11871afac5b6